asim_formatter.py

```diff
@@ -15,13 +15,12 @@ def _get_event_result(self, response: Response) -> str:
 
         return event_result
 
-    def _get_file_name(self, request: Request) -> str:
-        if not request.files:
-            return "N/A"
-        if len(request.files.keys()) == 1:
-            return list(request.files.keys())[0]
+    def _get_file_name(self, response: Response) -> str:
+        content_disposition = response.headers.get("Content-Disposition")
+        if content_disposition:
+            return content_disposition.split("filename=")[-1].strip('"')
```

Comment:
This will blow up if the `Content-Disposition` header doesn't contain the `filename=` substring - it's optional; see https://datatracker.ietf.org/doc/html/rfc6266#page-4 and https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Disposition#syntax. It'll also include any parameters that follow the filename, which is undesirable.

A regex might be safer and more specific. Something similar to this (untested):
```suggestion
            search_result = re.search("filename=(.*)[;]", content_disposition)
            if search_result:
                return search_result.group(1)
            else:
                return "N/A"
```
Perhaps add tests for the cases that the MDN article uses.